### PR TITLE
[DTM] - Complex property ordering

### PR DESCRIPTION
**Pull request by Vagdevi1998**
Documents complex property ordering.

### CODE REVIEW

File: content/dtm/asset-types-bundle/asset-properties.md [hunk 1]
```diff
@@ -278,7 +278,7 @@ In the **Asset properties** simple asset properties are listed with their type.
 
 #### Complex asset property {#complex-asset-property}
 
-A complex asset property consists of multiple key value pairs, meaning, it contains a root object followed by one or more key value pairs. You can select the complex asset property option by switching on the toggle. If you switch on the toggle, you must define each key value pair as well as providing a Type for each pair.
+A complex asset property consists of multiple key value pairs, meaning, it contains a root object followed by one or more key value pairs. You can select the complex asset property option by switching on the toggle. If you switch on the toggle, you must define each key value pair as well as providing a Type for each pair. You can arrange the properties in the required order simply by dragging and dropping them. You can expand all key value pairs at a time using **Expand all**. Alternatively, you can expand each key-value pair to see a specific property. When you add a new key value pair, it automatically expands.
```
Comment (lra-sag) [inline content/dtm/asset-types-bundle/asset-properties.md:281]:
```suggestion
A complex asset property consists of multiple key value pairs, meaning, it contains a root object followed by one or more key value pairs. You can select the complex asset property option by switching on the toggle. If you switch on the toggle, you must define each key value pair as well as providing a Type for each pair. You can arrange the properties in the required order by dragging and dropping them. You can expand all key value pairs at a time using **Expand all**. Alternatively, you can expand each key-value pair to see a specific property. When you add a new key value pair, it automatically expands.
```

File: static/images/dtm/custom-property/dtm-property-library-complex-property.png
Comment (lra-sag) [inline static/images/dtm/custom-property/dtm-property-library-complex-property.png:1]: Update screenshot to show one collapsed key and one expanded.

File: content/dtm/asset-types-bundle/asset-properties.md [hunk 2]
```diff
@@ -278,7 +278,7 @@ In the **Asset properties** simple asset properties are listed with their type.
 
 #### Complex asset property {#complex-asset-property}
 
-A complex asset property consists of multiple key value pairs, meaning, it contains a root object followed by one or more key value pairs. You can select the complex asset property option by switching on the toggle. If you switch on the toggle, you must define each key value pair as well as providing a Type for each pair.
+A complex asset property consists of multiple key value pairs, meaning, it contains a root object followed by one or more key value pairs. You can select the complex asset property option by switching on the toggle. If you switch on the toggle, you must define each key value pair as well as providing a Type for each pair. You can arrange the properties in the required order by dragging and dropping them. You can expand all key value pairs at a time using **Expand all**. Alternatively, you can expand each key-value pair to see a specific property. When you add a new key value pair, it automatically expands.
```
Comment (lra-sag) [inline content/dtm/asset-types-bundle/asset-properties.md:281]:
```suggestion
A complex asset property consists of multiple key value pairs, meaning, it contains a root object followed by one or more key value pairs. You can select the complex asset property option by switching on the toggle. If you switch on the toggle, you must define each key value pair as well as providing a Type for each pair. You can arrange the properties in the required order by dragging and dropping them. Hover over the row to see the drag icon. Click and drag the drag icon for reordering. 
You can expand all key value pairs at a time using **Expand all**. Alternatively, you can expand each key-value pair to see a specific property. When you add a new key value pair, it automatically expands.
```

**Review by lra-sag — CHANGES_REQUESTED** [review]

File: static/images/dtm/custom-property/dtm-property-library-complex-property.png
Comment (Vagdevi1998) [reply static/images/dtm/custom-property/dtm-property-library-complex-property.png:1]: Updated

File: content/dtm/asset-types-bundle/asset-properties.md [hunk 3]
```diff
@@ -278,7 +278,8 @@ In the **Asset properties** simple asset properties are listed with their type.
 
 #### Complex asset property {#complex-asset-property}
 
-A complex asset property consists of multiple key value pairs, meaning, it contains a root object followed by one or more key value pairs. You can select the complex asset property option by switching on the toggle. If you switch on the toggle, you must define each key value pair as well as providing a Type for each pair.
+A complex asset property consists of multiple key value pairs, meaning, it contains a root object followed by one or more key value pairs. You can select the complex asset property option by switching on the toggle. If you switch on the toggle, you must define each key value pair as well as providing a Type for each pair. You can arrange the properties in the required order by dragging and dropping them. Hover over the row to see the drag icon. Click and drag the drag icon for reordering.
```
Comment (BeateRixen) [inline content/dtm/asset-types-bundle/asset-properties.md:281]:
```suggestion
A complex asset property consists of multiple key value pairs, meaning, it contains a root object followed by one or more key value pairs. You can select the complex asset property option by switching on the toggle. If you switch on the toggle, you must define each key value pair as well as providing a type for each pair. You can arrange the properties in the required order by dragging and dropping them. Hover over the row to see the drag icon. Click and drag the drag icon for reordering. 
```

File: content/dtm/asset-types-bundle/asset-properties.md [hunk 4]
```diff
@@ -278,7 +278,8 @@ In the **Asset properties** simple asset properties are listed with their type.
 
 #### Complex asset property {#complex-asset-property}
 
-A complex asset property consists of multiple key value pairs, meaning, it contains a root object followed by one or more key value pairs. You can select the complex asset property option by switching on the toggle. If you switch on the toggle, you must define each key value pair as well as providing a Type for each pair.
+A complex asset property consists of multiple key value pairs, meaning, it contains a root object followed by one or more key value pairs. You can select the complex asset property option by switching on the toggle. If you switch on the toggle, you must define each key value pair as well as providing a Type for each pair. You can arrange the properties in the required order by dragging and dropping them. Hover over the row to see the drag icon. Click and drag the drag icon for reordering. 
+You can expand all key value pairs at a time using **Expand all**. Alternatively, you can expand each key-value pair to see a specific property. When you add a new key value pair, it automatically expands.
```
Comment (BeateRixen) [inline content/dtm/asset-types-bundle/asset-properties.md:282]: Not sure about the spelling of "key value pair". It is spelled with and without hyphen here. Please unify. Preferred spelling definitely is "key-value pair".

Comment (Vagdevi1998) [reply content/dtm/asset-types-bundle/asset-properties.md]: Modified to key-value pair

**Review by pavanbhosle — APPROVED** [review]

**Review by BeateRixen — APPROVED** [review]

**Review by lra-sag — APPROVED** [review]

**Pull request merged**